doc/tutorial/get_started.rst

```diff
@@ -183,6 +194,13 @@ Complete the following steps:
      lxc config device add micro3 eth1 nic network=microbr0 name=eth1
      lxc config device add micro4 eth1 nic network=microbr0 name=eth1
 
+#. If you created a `cephbr0` network, create and add network interfaces that use the dedicated Ceph network to each VM::
```

Comment:
```suggestion
#. If you created a ``cephbr0`` network, create and add network interfaces that use the dedicated Ceph network to each VM::
```

This will fix the spelling issue. ;)
In RST, one single quote gives italics, and you need two for code markup.